library/webpack-plugins/merge-css-plugin.js

```diff
@@ -94,37 +100,40 @@ module.exports = function mergeCssPlugin() {
               .map(({ cssHash }) => cssHash)
 
             chunkCssHashes.set(chunk, cssHashes)
-            compilation.applyPlugins('chunk-css-hashes', chunk.name, cssHashes)
+            compilation.hooks.chunkCssHashes.call(chunk.name, cssHashes)
           })
         })
 
         // make sure the __webpack_css_chunk_hashes__ is available in modules (code copied from ExtendedApiPlugin)
         compilation.dependencyFactories.set(ConstDependency, new NullFactory())
         compilation.dependencyTemplates.set(ConstDependency, new ConstDependency.Template())
-        compilation.mainTemplate.plugin('require-extensions', function(source, chunk, hash) {
+        compilation.mainTemplate.hooks.requireExtensions.tap(p, function(source, chunk, hash) {
```

Comment:
arrow function is fine

Reply:
In the last version we were forced to use `this`